torchtext/data/datasets_utils.py

```diff
@@ -246,10 +245,9 @@ def decorator(func):
                 argspec.args[1] == "split" and
                 argspec.varargs is None and
                 argspec.varkw is None and
-                len(argspec.kwonlyargs) == 0 and
-                len(argspec.annotations) == 0
+                len(argspec.kwonlyargs) == 0
                 ):
-            raise ValueError("Internal Error: Given function {} did not adhere to standard signature.".format(fn))
+            raise ValueError("Internal Error: Given function {} did not adhere to standard signature.".format(func))
```

Comment:
NIT: wondering if we want to rename `func` to be `fn` here to be consistent with the naming in the [`_wrap_split_argument_with_fn` method](https://github.com/pytorch/text/blob/97ee0c13db433bb0cb8a0876476618ffe642916b/torchtext/data/datasets_utils.py#L195)